File: blinkeye-framewk-comun-dominio/src/main/java/net/royal/spring/logistica/dominio/filtro/FiltroComunWhContrato.java
package net.royal.spring.logistica.dominio.filtro;

import net.royal.spring.framework.modelo.generico.DominioPaginacion;

public class FiltroComunWhContrato {
	private String compania;	
		
	private DominioPaginacion paginacion=new DominioPaginacion();
	public DominioPaginacion getPaginacion() {
		return paginacion;
	}

	public void setPaginacion(DominioPaginacion paginacion) {
		this.paginacion = paginacion;
	}

	public String getCompania() {
		return compania;
	}

	public void setCompania(String compania) {
		this.compania = compania;
	}
	
	
	
}

File: web-deteccion/src/app/royal/modulos/framework-comun/wh/dominio/filtro/FiltroComunWhContrato.ts
import { DominioPaginacion } from './../../../../../framework/modelo/generico/DominioPaginacion';
 

export class FiltroComunWhContrato {
    constructor(){
        this.paginacion = new DominioPaginacion();
		this.fechadoc_check=false
		this.fechaval_check=true
		this.tc=0
    }
    
    paginacion: DominioPaginacion;
    
     compania:string;
	 unidadnegocio:string;
	 clasificacioncontrato:string;
	 tipocontrato:string;
	 numerocontrato:string;
	 documentointerno:string;
	 modalidad:string;
	 clasificacion:string;
	 estado:string;
	 responsable:string;
	 fechadoc_desde:string;
	 fechadoc_hasta:string;
	 fechavalidez_desde:string;
	 fechavalidez_hasta:string;
	 proveedor:number;
	 moneda:string;
     busqueda:string
	 centrocostos_descri:string
	 centrocostos:string
	 tipoexportar:string
	 tc:number

	 //DATES
	 fechadoc_desde_date:Date;
	 fechadoc_hasta_date:Date;
	 fechavalidez_desde_date:Date;
	 fechavalidez_hasta_date:Date;
	 fechadoc_check:boolean
	 fechaval_check:boolean
    
}
